components/icons/ArrowIcon.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import Icon, { IconProps } from './Icon';
+
+interface Props extends IconProps {
+  animated?: boolean;
+  revert?: boolean;
+}
+
+export default function ArrowIcon({
+  animated = false,
+  fill = 'currentColor',
+  revert = false,
```

Comment:
NP: I'm not sure `revert` is intuitive here (I had to try it out to understand the intent). Maybe `leftToRight` would be more comprehensible?

Reply:
WDYT about `inverseDirection` ?